### DISCUSSION TITLE
Radeon ProRender USD for Blender 3.0

### DISCUSSION THREAD

**Discussion by bsavery** [GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon#308] [General]
With Blender 4.0, we contributed enhancements here back our USD Hydra rendering system directly to Blender's code ([more info](https://wiki.blender.org/wiki/Reference/Release_Notes/4.0/Add-ons)).  

The updated plugin uses that Hydra rendering system (and works only with Blender 4.0). 

Furthermore the RPR addon for Blender now allows collaboration of USD files across systems and applications via the new AMD [Render Studio](https://github.com/GPUOpen-LibrariesAndSDKs/RenderStudioKit/releases/tag/v1.0.0) system.

<hr /><em>This discussion was created from the release <a href='https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon/releases/tag/v3.0.0'>Radeon ProRender USD for Blender 3.0</a>.</em>

**Comment by squinonescolon** [comment]
This looks great, but I don’t see precompiled binaries for osx, specifically m1,etc. is there any hope of getting some? I’ve been waiting for this to make it to Blender main, but this would be a great intermediary step until we get the equivalent of a native USD stage in Blender.
